Microsoft.Toolkit.Uwp.UI.Controls.Graph/AadLogin/AadLogin.cs

```diff
@@ -78,7 +78,18 @@ public AadLogin()
         /// <returns>True if sign in successfully, otherwise false</returns>
         public async Task<bool> SignInAsync()
         {
-            if (await GraphService.TryLoginAsync())
+            var success = false;
+
+            try
+            {
+                success = await GraphService.TryLoginAsync();
+            }
+            catch (Exception ex)
+            {
+                SignInFailed?.Invoke(this, new SignInFailedEventArgs(ex));
```

Comment:
To clarify, if they hit 'cancel' this will be success = false but no exception?  Assuming yes, might be useful to document that workflow as well to say that this event will only fire if there's a problem during the sign-in process.

Reply:
Good call, updated the docs